src/components/Footer.tsx

```diff
@@ -61,4 +61,194 @@ const Footer = () => (
+		</svg>
+	</a>
+);
+
+const InstagramLink = () => (
+	<a
+		href="https://instagram.com/galsendev221"
+		rel="noreferrer"
+		target="_blank"
+		className="transition hover:opacity-75"
+	>
+		<span className="sr-only">Instagram</span>
+
+		<svg
+			className="w-6 h-6"
+			fill="#E1306C"
+			viewBox="0 0 24 24"
+			aria-hidden="true"
+		>
+			<path d="M12 2.163c3.204 0 3.584.012 4.85.07 3.252.148 4.771 1.691 4.919 4.919.058 1.265.069 1.645.069 4.849 0 3.205-.012 3.584-.069 4.849-.149 3.225-1.664 4.771-4.919 4.919-1.266.058-1.644.07-4.85.07-3.204 0-3.584-.012-4.849-.07-3.26-.149-4.771-1.699-4.919-4.92-.058-1.265-.07-1.644-.07-4.849 0-3.204.013-3.583.07-4.849.149-3.227 1.664-4.771 4.919-4.919 1.266-.057 1.645-.069 4.849-.069zm0-2.163c-3.259 0-3.667.014-4.947.072-4.358.2-6.78 2.618-6.98 6.98-.059 1.281-.073 1.689-.073 4.948 0 3.259.014 3.668.072 4.948.2 4.358 2.618 6.78 6.98 6.98 1.281.058 1.689.072 4.948.072 3.259 0 3.668-.014 4.948-.072 4.354-.2 6.782-2.618 6.979-6.98.059-1.28.073-1.689.073-4.948 0-3.259-.014-3.667-.072-4.947-.196-4.354-2.617-6.78-6.979-6.98-1.281-.059-1.69-.073-4.949-.073zm0 5.838c-3.403 0-6.162 2.759-6.162 6.162s2.759 6.163 6.162 6.163 6.162-2.759 6.162-6.163c0-3.403-2.759-6.162-6.162-6.162zm0 10.162c-2.209 0-4-1.79-4-4 0-2.209 1.791-4 4-4s4 1.791 4 4c0 2.21-1.791 4-4 4zm6.406-11.845c-.796 0-1.441.645-1.441 1.44s.645 1.44 1.441 1.44c.795 0 1.439-.645 1.439-1.44s-.644-1.44-1.439-1.44z" />
+		</svg>
+	</a>
+);
+
+const FacebookLink = () => (
+	<a
+		href="https://facebook.com/galsendev221"
+		rel="noreferrer"
+		target="_blank"
+		className="transition hover:opacity-75"
+	>
+		<span className="sr-only">Facebook</span>
+
+		<svg
+			className="w-6 h-6"
+			fill="#4267B2"
+			viewBox="0 0 24 24"
+			aria-hidden="true"
+		>
+			<path d="M12 0c-6.627 0-12 5.373-12 12s5.373 12 12 12 12-5.373 12-12-5.373-12-12-12zm3 8h-1.35c-.538 0-.65.221-.65.778v1.222h2l-.209 2h-1.791v7h-3v-7h-2v-2h2v-2.308c0-1.769.931-2.692 3.029-2.692h1.971v3z" />
+		</svg>
+	</a>
+);
+
+const YoutubeLink = () => (
+	<a
+		href="https://youtube.com/@GalsenDev221"
+		rel="noreferrer"
+		target="_blank"
+		className="transition hover:opacity-75"
+	>
+		<span className="sr-only">YouTube</span>
+
+		<svg
+			xmlns="http://www.w3.org/2000/svg"
+			viewBox="0 0 256 180"
+			className="w-6 h-6"
+			fill="#FF0000"
+			aria-hidden="true"
+		>
+			<path d="M250.346 28.075A32.18 32.18 0 0 0 227.69 5.418C207.824 0 127.87 0 127.87 0S47.912.164 28.046 5.582A32.18 32.18 0 0 0 5.39 28.24c-6.009 35.298-8.34 89.084.165 122.97a32.18 32.18 0 0 0 22.656 22.657c19.866 5.418 99.822 5.418 99.822 5.418s79.955 0 99.82-5.418a32.18 32.18 0 0 0 22.657-22.657c6.338-35.348 8.291-89.1-.164-123.134Z"></path>
+			<path
+				fill="#FFF"
+				d="m102.421 128.06l66.328-38.418l-66.328-38.418z"
+			></path>
+		</svg>
+	</a>
+);
+
```

Comment:
Since #66 we are using sprites for the SVGs. That why we're using the `SocialIcon` component for displaying the logos. We can delete these

Reply:
Yes, you're right, I left the "production" branch, that's why.